<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_page</class>
 <widget class="QWidget" name="search_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>576</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>书目检索</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>书目检索</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>105</y>
     <width>691</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 245, 240);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="year_section">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>42</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>1900</number>
   </property>
   <property name="maximum">
    <number>2017</number>
   </property>
   <property name="value">
    <number>2017</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="year_section_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>42</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="minimum">
    <number>1800</number>
   </property>
   <property name="maximum">
    <number>2017</number>
   </property>
   <property name="value">
    <number>2017</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="min_price">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>62</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="max_price">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>62</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="search_basis">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>书号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>类别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>出版社</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>年份</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>作者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>价格</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="search_input">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>60</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="search_basis_2">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>60</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>书号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>类别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>出版社</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>年份</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>作者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>价格</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>60</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>搜 索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_input_2">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>60</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="return_main">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>60</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="to">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
